Build cross docs (#98)
kubernetes-sigs/etcdadm: README.md

## Getting Started

### Quick start: How to quickly create a production etcd cluster using etcdadm

- Build a release of etcdadm.  Assuming your nodes are linux, run `GOOS=linux make container-build`.

> - copy the created `etcdadm` file it to every node on your cluster.

Reviewer: ```suggestion
- copy the created `etcdadm` file to every node on your cluster.
```

- Create a new cluster on your first node.
- On successive nodes, just run "Adding a member" instructions below.

### Building, Contributing

#### As a developer
- Setup Golang: The fastest way to do this is using https://github.com/travis-ci/gimme
- Then run `go get -u sigs.k8s.io/etcdadm`

#### As a user

> - Build in docker, its easy ! Just run `make container-build`

Reviewer: ```suggestion
- Build in docker, it's easy ! Just run `make container-build`
```

- The outputted binary will be natively runnable on your mac or linux machine.

### Contributing
- Follow the above directions, 

> - cd $GOPATH/github.com/platform9/etcdadm

Reviewer: ```suggestion
- cd $GOPATH/sigs.k8s.io/etcdadm
```

etcdadm init
```

You now have an etcd 'cluster' which is usable, of size 1.

### Step 2: Adding a member

> Although not necessary, its obviously a good idea to have more then 1 node in your cluster.

Reviewer: ```suggestion
Although not necessary, it's obviously a good idea to have more than 1 node in your cluster.
```
